dot/core/service_test.go

```diff
@@ -287,9 +287,9 @@ func TestHandleChainReorg_WithReorg_Transactions(t *testing.T) {
 	addTestBlocksToState(t, height, s.blockState.(*state.BlockState))
 
 	// create extrinsic
-	ext := extrinsic.NewIncludeDataExt([]byte("nootwashere"))
-	tx, err := ext.Encode()
+	enc, err := scale.Marshal([]byte("nootwashere"))
 	require.NoError(t, err)
+	tx := append([]byte{2}, enc...)
```

Comment:
Am I right in saying that the 2 is appended because its a IncludeDataType extrinsic? If so, since we are deleting that file, will we be able to know what the leading byte represents/what values it can have in the future? I was initially confused at what this 2 was and I'm not sure how I would figure that out with the extrinsic file gone. Maybe this isn't a huge concern but I wanted to ask

Reply:
yeah would be good to document this

Reply:
I'll add documentation

Reply:
Are there other possible prefixes that should be documented besides the 2 case? Or can those be found elsewhere if need be?